WebContent/WEB-INF/dox/de/collectInactiveMsg.htm

```diff
@@ -0,0 +1,23 @@
+<!--
+    Mango - Open Source M2M - http://mango.serotoninsoftware.com
+    Copyright (C) 2006-2011 Serotonin Software Technologies Inc.
+    @author Matthew Lohbihler
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see http://www.gnu.org/licenses/.
+ -->
```

Comment:
In my opinion we should not include that comment section inside help messages. Copyrights are defined in the licence file in the root directory.  Especial if here as author was annotated Matthew Lohbihler and Copyrights to Serotonin software. Without this comment messages will be more readable.